ReactAndroid/src/main/java/com/facebook/react/uimanager/events/ContentSizeChangeEvent.java

```diff
@@ -16,12 +18,26 @@
   private final int mWidth;
   private final int mHeight;
 
+  /**
+   * See {@link Event#Event(int)}.
+   *
+   * @param viewTag
+   * @param width
+   * @param height
+   */
+  @Deprecated
```

Comment:
Do you have an idea how we could make events and tags work without patching any of them?